designs/setdict-design.md

```diff
@@ -0,0 +1,104 @@
+# Set/Tuple/Dictionary: Project Milestone For Week 7
+For now, we'd like to start with dictionary. The following design document will be based on dictionary, but may be subject to change later.
+
+## Test Cases
+1. Initialize the empty dictionary in Chocopy.   
+```python
+dict_1 : [int, int] = None 
+dict_1 = {}
+```
+
+2. Store a value in the dictionary given a key.  
+```python 
+dict_1 : [int, int] = None
+dict_1 = {}   
+dict_1[1] = 2
+```
+
+3. Extract the value from the dictionary given a key that exists in the dictionary.  
+```python 
+dict_1 : [int, int] = None 
+dict_1 = {} 
+dict_1[1] = 2   
+print(dict_1[1])  # should output 2
+```
+
+4. Obtain the type of the dictionary object. 
+```python
+dict_1 : [int, int] = None 
+dict_1 = {} 
+print(type(dict_1))  # should output "dict"
+```
+
+5. Get the value from the dictionary using the method get(). Return None if doesn't exist.
+```python
+dict_1 : [int, int] = None 
+dict_1 = dict({1:2, 3:4})

```

Comment:
This is interesting syntax. What's the type of `{1: 2, 3: 4}`? Why wouldn't you initialize `dict_1 = {1: 2, 3: 4}`?